README.md

```diff
@@ -128,3 +129,4 @@ For more details please check [violines/rest-bundle Wiki](https://github.com/vio
 1. cd docker/
 1. pull latest image(s): docker-compose pull
 1. create the container(s): docker-compose up -d
+    - To run the tests in one of the containers, use `docker-compose exec php80 /app/vendor/bin/phpunit`
```

Comment:
You can also run cs-check, phpunit and psalm with
`composer test`

I think that fits better here